packages/flutterfire_ui/test/table_builder_test.dart

```diff
@@ -0,0 +1,182 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:fake_cloud_firestore/fake_cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutterfire_ui/src/firestore/table_builder.dart';
+
+Future<void> main() async {
+  final instance = FakeFirebaseFirestore();
+  final collection = instance.collection('persons').withConverter<Person>(
+        fromFirestore: (snapshot, options) => Person.fromMap(snapshot.data()!),
+        toFirestore: (data, option) => data.toMap(),
+      );
+
+  final bob = Person(
+    firstName: 'Bob',
+    address: Address(street: 'Awesome Road', city: 'FlutterFire City'),
+  );
+  await collection.add(bob);
+
+  testWidgets('TableBuilder WITHOUT CelBuilder is render as expected',
+      (WidgetTester tester) async {
+    // Create the widget by telling the tester to build it.
```

Comment:
This comment seems redundant and is present twice in the file. I'd remove it

Reply:
Will do